lib/adal-angular.js

```diff
@@ -264,7 +264,8 @@
                                 if (!_oauthData.isAuthenticated) {
                                     if (!_adal._renewActive && !_adal.loginInProgress()) {
                                         _adal.info('State change event for:' + $location.$$url);
-                                        loginHandler();
+                                        var $state = $injector.get('$state');
+                                        loginHandler(window.location.origin + '/' + $state.href(toState, toParams));
```

Comment:
We do not need to use the window.origin property here. Please change this to:  
var loginStartPage = $state.href(toState, toParams, { absolute: true })
loginHandler(loginStartPage);